### DISCUSSION TITLE
2024년 01월 16일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#105] [General]
# 2024년 01월 16일

TODO 연구실 미팅 Prometheus + Grafana 서버 설정 스픽 듀오링고 1시간 Suits 한 화 보기 LOG 연구실 미팅에 갔다왔다. 오늘도 내가 할 건 딱히 없었다. 오늘도 연구실 가는 길에 지하철을 잘못 탔다. 5호선 타야 되는데 8호선 탔다. 다행히 석촌에서 내려서 9호선으로 갈아탈 수 있었다. 천만 다행이다!! S-HOOK 도메인 회의…

https://vero.wiki/2024-01-16/

**Comment by ukkodeveloper** [comment]
오늘의 영어 ㅋㅋㅋ 멋지다

**Reply by Cyma-s** [reply]
영어 잘하기 9개월의 전사 가보자
